rest/RESTClient.ts

```diff
@@ -63,11 +70,21 @@ export default class RESTClient {
         const discordBucket = api.headersIn['x-ratelimit-bucket'] as string | undefined;
         if (api.headersIn['x-ratelimit-global'] === 'true') {
           this.globallyRateLimited = true;
+
           setTimeout(() => {
             this.globallyRateLimited = false;
+
+            while (this.queue.length >= 1) {
+              const _request = this.queue.shift();
+
+              // @ts-ignore ts(2722) "Cannot invoke an object which is possibly 'undefined'."
+              _request();
+            }
           }, api.json.retry_after * 1e3);
+          // eslint-disable-next-line no-undefined
         } else if (discordBucket !== undefined) {
           const bucket = this.rateLimits.getBucket(discordBucket) || this.rateLimits.get(rateLimitRoute) as RESTBucket;
+          // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
This shouldn't be needed? Is it showing up as a warning or error?

Reply:
an error
`Expected an assignment or function call and instead saw an expression.`

Reply:
why not just check `if (!value)`?

Reply:
Alright, I just did that.